README.md

```diff
@@ -33,7 +33,7 @@ This example is compatible with [this version of a small Rails app with Action C
 
 
 The available hooks to tie in to are:
- - `disconnected {}`
+ - `disconnected { |code, reason| }`
```

Comment:
it's been a while for me, but where would someone find out what possible codes and reasons could be passed here?

Reply:
I have no idea! The last one I had was `reason` being `AuthError`, I don't remember the `code`.

I guess you can find it from the ActionCable protocol specification, but Is that important for this PR?

Reply:
nah, thanks! It's just been ages since I've worked with Rails' things

Reply:
👍